### PR TITLE
text file

### PR DESCRIPTION
**Pull request by CSGLMZBA**

### CODE REVIEW

File: text.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Esto es un archivo local
```
Comment (Arthyom) [inline text.txt:1]: Esto no compila, por favor reviselo

Comment (CSGLMZBA) [reply text.txt:1]: :c

**Comment by CSGLMZBA** [comment]
can't be merged

File: text.txt (hunk 1 above)
Comment (Arthyom) [reply text.txt:1]: Ya estan resueltos

**Pull request closed without merging**